M/MONA/build_tarballs.jl

```diff
@@ -0,0 +1,42 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "MONA"
+version = v"1.4.18"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("http://www.brics.dk/mona/download/mona-1.4-18.tar.gz",
+                  "ece10e1e257dcae48dd898ed3da48f550c6b590f8e5c5a6447d0f384ac040e4c"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/mona*
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+extra=""
+if [[ "${target}" == *-apple-* ]]; then
+  extra="LDFLAGS=\"-Wl,-undefined dynamic_lookup\""
```

Comment:
This syntax is incorrect.  I don't know exactly what flag you want to pass to the linker (and the syntax for that is also wrong), but in the first instance you need to remove all the `\"`.